lib/sidekiq_rx_impl.cc

```diff
@@ -256,9 +318,10 @@ void sidekiq_rx_impl::set_rx_filter_override_taps(const std::vector<float> &taps
 }
 
 void sidekiq_rx_impl::set_rx_frequency(double value) {
-	if (set_frequency(value)) {
-		tag_now = true;
-	}
+
```

Comment:
I'm confused by this update...it looks like you removed the check for success and are always tagging?

Reply:
So the base method set_frequency() either exits on error or returns 0, so I don't think tag_now ever got set.

As an aside, can you explain tags to me?  I don't get those or how to use them in a gnuradio-companion application.

I attempted to spend some time playing with them but an overview would be helpful.

Reply:
Ok, I think I understand now.

The code is designed that if any of the fields in the tag change, it will set tag_now = true to force tags to be sent on the next work() call.  The old code would do the if on the call to set_frequency but that doesn't return anything but 0 so, I cleaned it up and always set tag_now = true.

Reply:
Ok, so you would prefer that I remove any this->stop() calls.  Then when I am doing a read_ call, it returns a value that I don't have.  Just return some default?